src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27922.cs

```diff
@@ -0,0 +1,26 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue27922 : _IssuesUITest
+{
+	public override string Issue => "[WinUI]CollectionView with GroupHeader enabled should scroll properly with groupheader";
+
+	public Issue27922(TestDevice device) : base(device)
+	{
+	}
+
+	[Test]
+	[Category(UITestCategories.CollectionView)]
+	public void ItemShouldbeScrolledbasedOnGroupHeader()
```

Comment:
Pending snapshots on Windows. Already available in the latest build:
![image](https://github.com/user-attachments/assets/d7a88640-fd85-406a-818a-0775f7293c08)

Could you commit it?

Reply:
Hi @jsuarezruiz  ,I have added the snapshot. Please let me know if you have any concerns.